symbolic/division-by-zero-3-35.c

```diff
@@ -14,7 +14,7 @@ uint64_t main() {
 
   x = malloc(8);
 
-  read(1, x, 1);
+  read(0, x, 1);
```

Comment:
Please check with @ChrisEdel if this is ok.

Reply:
Seems fine.

Reply:
Why did you use 0?

Reply:
What's the difference actually between 0 and 1?

Reply:
0 = stdin; 1 = stdout; 2 = stderror